pandas/core/indexing.py

```diff
@@ -834,11 +831,9 @@ def _getitem_nested_tuple(self, tup: Tuple):
             if self.name != "loc":
                 # This should never be reached, but lets be explicit about it
                 raise ValueError("Too many indices")
-            try:
+            with suppress(IndexingError):
                 result = self._handle_lowerdim_multi_index_axis0(tup)
                 return result
```

Comment:
You could avoid the declaration of result here.

Reply:
Thank you! Changed.